roles/nvidiadl_ssd_benchmark_deploy/tasks/main.yml

```diff
@@ -0,0 +1,15 @@
+---
+- name: Create the entrypoint ConfigMap
+  command: 
+    oc create cm custom-config-script --from-file="{{ entrypoint }}"
+
+- name: Deploy pod to run Nvidia Deep Learning SSD Detection benchmark and download data
+  command: 
+    oc apply -f {{ nvidiapod }}
+
+# - name: Store the logs of the Nvidia Deep Learning SSD Training benchmark
+#   shell:
+#     oc logs pod/nvidiadl -n default | grep -o "[^$(printf '\r')]*$"
+#        -oyaml
+#        -n openshift-operators
+#        > {{ artifact_extra_logs_dir }}/nvidiadl-bench.yml
```

Comment:
you'll need to wait for the pod, see in gpu-burn or elsewhere, we already have tasks for it
search for `completed|failed|error`, it's something similar you'll want (ie, don't wait with the logs but check the Pod status)